common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1824,6 +1828,30 @@ object CelebornConf extends Logging {
       .doc("Threads number of message dispatcher event loop for roles")
       .fallbackConf(RPC_DISPATCHER_THREADS)
 
+  val RPC_SLOW_THRESHOLD: ConfigEntry[Long] =
+    buildConf("celeborn.rpc.slow.threshold")
+      .categories("network")
+      .doc("threshold for RPC framework to log slow RPC")
+      .version("0.6.0")
+      .longConf
+      .createWithDefault(1000000000)
+
+  val RPC_SLOW_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.rpc.slow.interval")
+      .categories("network")
+      .doc("min interval (ms) for RPC framework to log slow RPC")
+      .version("0.6.0")
+      .longConf
+      .createWithDefault(-1)
+
+  val RPC_TRACE_DUMP_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.rpc.dump.interval")
+      .categories("network")
+      .doc("min interval (ms) for RPC framework to dump performance summary")
+      .version("0.6.0")
+      .longConf
+      .createWithDefault(60000)
+
```

Comment:
```suggestion
  val RPC_SLOW_THRESHOLD: ConfigEntry[Long] =
    buildConf("celeborn.rpc.slow.threshold")
      .categories("network")
      .doc("threshold for RPC framework to log slow RPC")
      .version("0.6.0")
      .timeConf(TimeUnit.NANOSECONDS)
      .createWithDefaultString("1s")

  val RPC_SLOW_INTERVAL: OptionalConfigEntry[Long] =
    buildConf("celeborn.rpc.slow.interval")
      .categories("network")
      .doc("min interval (ms) for RPC framework to log slow RPC")
      .version("0.6.0")
      .timeConf(TimeUnit.MILLISECONDS)
      .createOptional

  val RPC_TRACE_DUMP_INTERVAL: ConfigEntry[Long] =
    buildConf("celeborn.rpc.dump.interval")
      .categories("network")
      .doc("min interval (ms) for RPC framework to dump performance summary")
      .version("0.6.0")
      .timeConf(TimeUnit.MILLISECONDS)
      .createWithDefaultString("60s")
```

Reply:
sure, will post a new commit to address those comments